custom_components/ha-google-home/api.py

```diff
@@ -104,10 +116,26 @@ def get_google_devices_information(self):
                             device=device.device_name, error=API_RETURNED_UNKNOWN
                         )
                     )
+            else:
+                offline_devices.append(device.device_name)
 
             device.timers = timers
             device.alarms = alarms
             _LOGGER.debug(device)
 
+        """Gives the user a warning if the device is offline, but will not remove entities or device from
+        HA device registry"""
```

Comment:
The comment inside of method rarely is done as docstring `""" ... """`, even if it's a multiline comment, let's just keep it as a comment `# ....`